hoomd/md/EvaluatorPairLJ0804.h

```diff
@@ -0,0 +1,233 @@
+
+    If needsDiameter() returns true, a setDiameter(Scalar di, Scalar dj) method will be called to set the two diameters.
+    Similarly, if needsCharge() returns true, a setCharge(Scalar qi, Scalar qj) method will be called to set the two
+    charges.
+
+    All other arguments are common among all pair potentials and passed into the constructor. Coefficients are handled
+    in a special way: the pair evaluator class (and PotentialPair) manage only a single parameter variable for each
+    type pair. Pair potentials that need more than 1 parameter can specify that their param_type be a compound
+    structure and reference that. For coalesced read performance on G200 GPUs, it is highly recommended that param_type
+    is one of the following types: Scalar, Scalar2, Scalar4.
+
+    The program flow will proceed like this: When a potential between a pair of particles is to be evaluated, a
+    PairEvaluator is instantiated, passing the common parameters to the constructor and calling setDiameter() and/or
+    setCharge() if need be. Then, the evalForceAndEnergy() method is called to evaluate the force and energy (more
+    on that later). Thus, the evaluator must save all of the values it needs to compute the force and energy in member
+    variables.
+
+    evalForceAndEnergy() makes the necessary computations and sets the out parameters with the computed values.
+    Specifically after the method completes, \a force_divr must be set to the value
+    \f$ -\frac{1}{r}\frac{\partial V}{\partial r}\f$ and \a pair_eng must be set to the value \f$ V(r) \f$ if \a energy_shift is false or
+    \f$ V(r) - V(r_{\mathrm{cut}}) \f$ if \a energy_shift is true.
+
+    A pair potential evaluator class is also used on the GPU. So all of its members must be declared with the
+    DEVICE keyword before them to mark them __device__ when compiling in nvcc and blank otherwise. If any other code
+    needs to diverge between the host and device (i.e., to use a special math function like __powf on the device), it
+    can similarly be put inside an ifdef __HIPCC__ block.
+
+    <b>LJ specifics</b>
+
+    EvaluatorPairLJ0804 evaluates the function:
+    \f[ V_{\mathrm{LJ}}(r) = 4 \varepsilon \left[ \left( \frac{\sigma}{r} \right)^{8} -
+                                           \left( \frac{\sigma}{r} \right)^{4} \right] \f]
+    broken up as follows for efficiency
+    \f[ V_{\mathrm{LJ}}(r) = r^{-4} \cdot \left( 4 \varepsilon \sigma^{8} \cdot r^{-4} -
+                                            4 \varepsilon \sigma^{4} \right) \f]
+    . Similarly,
+    \f[ -\frac{1}{r} \frac{\partial V_{\mathrm{LJ}}}{\partial r} = r^{-2} \cdot r^{-4} \cdot
+            \left( 8 \cdot 4 \varepsilon \sigma^{8} \cdot r^{-4} - 4 \cdot 4 \varepsilon \sigma^{4} \right) \f]
+
+    The LJ potential does not need diameter or charge. Two parameters are specified and stored in a Scalar2. \a lj1 is
```

Comment:
```suggestion
    The LJ potential does not need diameter or charge. \a lj1 is
```
The code does not use a Scalar2.